sotodlib/preprocess/processes.py

```diff
@@ -1760,6 +1760,23 @@ def process(self, aman, proc_aman):
         if self.process_cfgs:
             pointing_model.apply_pointing_model(aman)
 
+class IIRParamsCorrection(_Preprocess):
+    """Correct missing iir_params
+
+    Example config block::
+
+        - name : "iir_params_correction"
+          process: True
```

Comment:
Can you add a "time_range" option, e.g.
```
time_range: ["2023-01-01", "2025-01-01"]
```

As discussed (#969), I don't like just painting over missing metadata, assuming that any failures in the future will be the same cause as failures of the past.  If we believe that all SATP3 filter issues in the past have been just reporting errors, but the filter was configured correctly, then we can _explicitly_ say "it's safe to patch these data by just writing in default values".